geometry_msgs/PoseStamped target
---
bool ok
string message
